admin/packaging.py

```diff
@@ -857,6 +857,10 @@ def omnibus_package_builder(
         steps=(
             InstallVirtualEnv(virtualenv=virtualenv),
             InstallApplication(virtualenv=virtualenv,
+                               package_uri='pip==8.1.1'),
```

Comment:
Sorry side thought here.

Any chance that if we just update pip or virtualenv in the docker image, we wouldn't have to do any of this?